build.assets/robotest/pr_config.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source $(dirname $0)/utils.sh
+
+# UPGRADE_MAP maps gravity version -> list of linux distros to upgrade from
+declare -A UPGRADE_MAP
+UPGRADE_MAP[$(recommended_upgrade_tag $(branch 7.0.x))]="centos:7 debian:9 ubuntu:18" # compatible LTS version
```

Comment:
Did we execute all 3 OS'es on each PR before as well, or is that a new thing?

Reply:
This was an unintentional change resulting from cut & paste from nightly at some point during the refactoring..  I will revert back to only ubuntu:18.

I think we probably test some additional OSes on PR, but I'd recommend those as installs / wrapped into expand/shrink instead of as upgrade (which is typically our heaviest scenario).

Reply:
Removed in a89b054.